### DISCUSSION TITLE
Need Guidance

### DISCUSSION THREAD

**Discussion by ghost** [aravindvnair99/AravindVNair99#7] [Q&A] [answered]
Hello Big Brother I am Ashwani  from Jharkhand.
Currently I now java(DSA) ,html, css (bootstrap),and learning javascript and react. As I live in a small Village so I don't have any mate or any exposure to tech environment to learn in a correct way and do Open source Contribution I want to ask  can you mentor me it will be very helpful if you guide me.

**Comment by aravindvnair99** [comment, marked as answer]
Hey @mrkool00!

Check out the following:

1. https://www.freecodecamp.org
2. https://summerofcode.withgoogle.com/
3. https://opensource.guide/
4. https://www.codetriage.com/

**Reply by ghost** [reply]
Thank you ☺️

**Reply by aravindvnair99** [reply]
Happy to help! :)
